From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Sat, 22 Aug 2020 14:31:59 +0200
Subject: Rollback dav1d upgrades that break x86

Revert "Roll src/third_party/dav1d/libdav1d/ 54f92068c..47daa4df3 (1 commit)"

This reverts commit e6447f7294588dcab826736c267cdab094a1643e.

Revert "Roll src/third_party/dav1d/libdav1d/ 07261e8c3..54f92068c (15 commits)"

This reverts commit b47516cde6a4d570b4c29b761d31c26009f9611f.

Revert "Roll src/third_party/dav1d/libdav1d/ b585f051c..07261e8c3 (39 commits)"

This reverts commit d723c1391cd7f73dedfd97805f1503010edf06ad.
---
 DEPS                                  | 2 +-
 third_party/dav1d/dav1d_generated.gni | 4 +---
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/DEPS b/DEPS
--- a/DEPS
+++ b/DEPS
@@ -774,7 +774,7 @@ deps = {
     Var('chromium_git') + '/angle/angle.git' + '@' +  Var('angle_revision'),
 
   'src/third_party/dav1d/libdav1d':
-    Var('chromium_git') + '/external/github.com/videolan/dav1d.git' + '@' + '47daa4df3387c1529536243db80b81f3878cbf76',
+    Var('chromium_git') + '/external/github.com/videolan/dav1d.git' + '@' + 'b585f051c365bbada36bd066dd0a8a237fbf9dca',
 
   'src/third_party/dawn':
     Var('dawn_git') + '/dawn.git' + '@' +  Var('dawn_revision'),
diff --git a/third_party/dav1d/dav1d_generated.gni b/third_party/dav1d/dav1d_generated.gni
--- a/third_party/dav1d/dav1d_generated.gni
+++ b/third_party/dav1d/dav1d_generated.gni
@@ -20,7 +20,7 @@ x86_asm_sources = [
   "libdav1d/src/x86/looprestoration.asm",
   "libdav1d/src/x86/looprestoration_ssse3.asm",
   "libdav1d/src/x86/mc.asm",
-  "libdav1d/src/x86/mc_sse.asm",
+  "libdav1d/src/x86/mc_ssse3.asm",
   "libdav1d/src/x86/msac.asm",
 ]
 
@@ -37,11 +37,9 @@ x86_template_sources = [
 arm32_asm_sources = [
   "libdav1d/src/arm/32/cdef.S",
   "libdav1d/src/arm/32/ipred.S",
-  "libdav1d/src/arm/32/itx.S",
   "libdav1d/src/arm/32/loopfilter.S",
   "libdav1d/src/arm/32/looprestoration.S",
   "libdav1d/src/arm/32/mc.S",
-  "libdav1d/src/arm/32/msac.S",
   "libdav1d/src/arm/32/util.S",
 ]
 
-- 
2.17.1

